kili/mutations/project/queries.py

```diff
@@ -154,10 +150,7 @@
 '''
 
 GQL_PROJECT_DELETE_ASYNCHRONOUSLY = '''
-mutation($projectID: ID!) {{
-  data: deleteProjectAsynchronously(where: {{
-      id: $projectID
-    }}
-    )
-}}
+mutation($where: ProjectWhere!) {
+  data: deleteProjectAsynchronously(where: $where)
```

Comment:
Did you tested this fragment ? Not sure if it would work, I think you need to return something (+ retrieving the token is useful for people turning a project public